src/cal.py

```diff
@@ -24,6 +24,24 @@
 from datetime import datetime
 
 
-class calender
+def makecalendar(month=1, year=2019):
+    cal = calendar.TextCalendar(calendar.SUNDAY)
+    formatted = cal.formatmonth(year, month)
+    return formatted
+
+
+date = input(f'Input a date in form month year:  ').split()
```

Comment:
Nice. Way to use input. You can also do this assignment very easily by checking the lenght of sys.argv but this is a much more robust way to handle your inputs. 

That said you're asking for the inputs backwards so please be a little more careful.